src/View/View.php

```diff
@@ -129,6 +129,7 @@ class View implements EventDispatcherInterface
      * The name of the subfolder containing templates for this View.
      *
      * @var string
+     * @deprecated 3.7.0 The property will become protected in 4.0.0. Use getTemplatePath()/setTemplatePath() instead.
```

Comment:
Should these become protected now and use `__get/__set/__isset` to emit deprecation warnings?

Reply:
I thought we weren't going to add more explicit warnings in `3.7`, but if we are then yes could make them protected and add magic methods for dealing with them.

Reply:
If we don't have hard warnings I don't think we can should make the breaking change in 4.0.0. I've been thinking more and I think 3.7 could be a 'catch up' release that contains a final sweep for deprecations.

Reply:
Okay, I'll update the PR.